src/pages/general/notFound/components/body404.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+
+export const Body404 = () => {
+  return (
+    <div className="panel-body">
+      <p>The page you are looking for might have been removed, had its name changed, or is temporarily unavailable. Please try the following:</p>
+      <ul className="list-group">
+        <li className="list-group-item">Make sure that the Web site address displayed in the address bar of your browser is spelled and formatted correctly.</li>
+        <li className="list-group-item">If you reached this page by clicking a link,
+          <a href="#"><b>contact us</b></a> to alert us that the link is incorrectly formatted.</li>
```

Comment:
Add a mailTo contact us, email address: info@lemoncode.net  

This is just to avoid that in the future we forget to add a real address.